src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -329,6 +329,26 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
             WHERE person_oversikt_status_id IN (select id from person_oversikt_status where fnr=?)
             ORDER BY tildelt_dato DESC
             """
+
+        const val AKTIV_OPPGAVE_WHERE_CLAUSE =
+            """
+        (
+        motebehov_ubehandlet = 't' 
```

Comment:
Litt urelatert til PR, men hva står `t` for?

Reply:
En annen måte å skrive `true` på i postgres - https://neon.tech/docs/data-types/boolean
Var vel noen som begynte med det her og så har man bare fortsatt :sweat_smile:

Reply:
Oh.. det gir for så vidt mening. Trodde kanskje det sto for en slags status.